Tests/js/views/assets/ez-navigationhubview-tests.js

```diff
@@ -152,7 +148,7 @@ YUI.add('ez-navigationhubview-tests', function (Y) {
                 userProfileView: this.userProfileView,
                 container: '.container',
                 user: this.userMock,
-                studioplusNavigationItems: [
+                studioNavigationItems: [
```

Comment:
I think, the whole property (and related tests) can be removed, unless you want to test any other tab - the studio tab in this case. Do you agree?

Reply:
removing it I think broke some of the tests, as some expects several zones, so hence the adaption here in the end result.